### PR TITLE
RELATED: BB-1692 Insight filtering by percentage measure value example

### PR DESCRIPTION
**Pull request by xMort**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by xMort** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1188.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4874/ : SUCCESS in 14m 17s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/183/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/184/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/1969/ : SUCCESS in 18m 29s

**Comment by xMort** [comment]
[https://gdc-live-examples-1188.herokuapp.com/hidden/measure-value-filter](https://gdc-live-examples-1188.herokuapp.com/hidden/measure-value-filter)
Note that you cannot check the filters function as these run against production platform servers.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4878/ : SUCCESS in 11m 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/188/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4879/ : FAILURE in 5m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/189/ : SUCCESS in 6s

File: examples/src/routes/MeasureValueFilter.jsx [hunk 1]
```diff
@@ -1,20 +1,68 @@
...
         <hr className="separator" />
-        <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        <h2>Comparison and range filters</h2>
+        <p>Example of filtering of visualization by either one or two measure values.</p>
+        <div className={"s-measure-value-filter-example-1"}>
+            <ExampleWithSource for={MeasureValueFilterExample} source={MeasureValueFilterExampleSRC} />
+        </div>
+        <h2>Filter by measure value shown in %</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is shown in %,{" "}
```
Comment (tomassatinsky) [inline examples/src/routes/MeasureValueFilter.jsx]: I would rephrase it to: When visualization is filtered by a measure that is shown in %, the filter value is in the original measure scale and not in the percentage scale that is displayed in the visualization.

File: examples/src/routes/MeasureValueFilter.jsx [hunk 2]
```diff
@@ -1,20 +1,68 @@
...
+        </p>
+        <div className={"s-measure-value-filter-example-2"}>
+            <ExampleWithSource
+                for={MeasureValueFilterShownInPercentageExample}
+                source={MeasureValueFilterShownInPercentageExampleSRC}
+            />
+        </div>
+        <h2>Filter by measure value stacked to 100%</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is stacked to 100%,{" "}
```
Comment (tomassatinsky) [inline examples/src/routes/MeasureValueFilter.jsx]: Similarly here: When visualization is filtered by a measure that is stacked to 100%, the filter value is in the original measure scale and not in the percentage scale that is displayed in the visualization.

File: examples/src/routes/MeasureValueFilter.jsx [hunk 3]
```diff
@@ -1,20 +1,68 @@
...
+        </p>
+        <div className={"s-measure-value-filter-example-3"}>
+            <ExampleWithSource
+                for={MeasureValueFilterStackedToHundredPercentExample}
+                source={MeasureValueFilterStackedToHundredPercentExampleSRC}
+            />
+        </div>
+        <h2>Filter by measure value formatted in %</h2>
+        <p>
+            In the case when visualization is filtered by a measure that is formatted in %,{" "}
```
Comment (tomassatinsky) [inline examples/src/routes/MeasureValueFilter.jsx]: ...and here: When the visualization is filtered by a measure that is formatted in %, the filter value is in the form of a ratio (for instance 0.5 which is 50%) and not in the percentage scale. <the rest is OK>

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4884/ : FAILURE in 3m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/195/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4885/ : SUCCESS in 6m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/197/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2570/ : SUCCESS in 12m 46s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/4886/ : SUCCESS in 4m 16s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1223/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/1603/ : SUCCESS in 18m 42s
